pages/static/dist/css/main.css

```diff
@@ -223,8 +223,8 @@ th {
   .navbar {
     display: none; }
   .btn > .caret, .form-wrapper #can_embed_form input[type=submit] > .caret,
-  .dropup > .btn > .caret, .form-wrapper #can_embed_form
```

Comment:
Why so many changes to css?

Reply:
@cjmabry the version upgrade for `gulp-sass` resulted in some refactors to how css is compiled. hopefully should be no regressions in UI styles.